tests/cmd/check-markdown/Makefile

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2017-2019 Intel Corporation
```

Comment:
@amshinde will update, I also need to include the authors for earlier static checks commits. Do you think I should separate out each tool? It might make it easier to divide up the authors

Reply:
I'll leave that you. I am ok with having all the tools in one commit and including all authors in that commit or have a separate commit for each tool. Either way would work, if you think you separating out each tool would make things easier, lets do that.

Reply:
@amshinde updated